CONTRIBUTING.md

```diff
@@ -0,0 +1,236 @@
+# Solid Technical Reports Contributing Guide
+
+Thank you for investing your time into contributing to the Solid project!
+
+This repository contains the source code of the work on [Solid Technical
+Reports](https://solidproject.org/TR/) (TR) of the [W3C Solid Community
+Group](https://www.w3.org/community/solid/) (CG) to meet the needs of the [Solid
+Project](https://solidproject.org/).
+
+The TRs include specifications, use cases and requirements, best practices and
+guidelines, primers and notes about the Solid ecosystem.
+
+
+## Code of conduct
+
+Read the [Solid Code of
+Conduct](https://github.com/solid/process/blob/main/code-of-conduct.md) and the
+[Positive Work Environment at W3C: Code of Ethics and Professional
+Conduct](https://www.w3.org/Consortium/cepc/) to keep our community approachable
+and respectable.
+
+
+## Contributions
+
+All substantive contributors must be members of the CG. It’s easy to [join the
```

Comment:
Maybe reverse that first sentence? "In order to be a substantive contributor of the spec, you must be a member of the CG."